index.html

```diff
@@ -492,6 +492,11 @@ <h3>Supported States and Properties</h3>
 			<h3>Inherited States and Properties</h3>
 			<p>Informative list of properties that are inherited onto a <a>role</a> from superclass roles. <a>States</a> and <a>properties</a> are inherited from superclass roles in the role <a>taxonomy</a>, not from ancestor <a>elements</a> in the <abbr title="Document Object Model">DOM</abbr> tree. These properties are not explicitly defined on the role, as the inheritance of properties is automatic. This information is provided to facilitate reading of the specification. The set of supported states and properties combined with inherited states and properties forms the full set of states and properties supported by the role.</p>
 		</section>
+		<section id="prohibitedattributes">
+				<h3>Prohibited States and Properties</h3>
+				<p>List of properties that are prohibited on a <a>role</a>. Authors MUST NOT specify a prohibited state or property.</p>
```

Comment:
Change "List of properties"
to "List of states and properties"
(for consistency).